archive-friend.php

```diff
@@ -46,7 +46,7 @@
             </nav>
 
           </div>
-          <div class="col s-col10 border-left border-right">
+          <div class="grid-item item-s-24 item-m-10 border-left border-right">
             <a href="<?php the_permalink() ?>"><?php the_post_thumbnail('col10-5to4'); ?></a>
```

Comment:
As they all refer to `l` is this really worth doing? I was just about to do it but I don't feel like there is much point. Yes we need a convention for the future but in this project we can assume `l` (as I always would) and then if needed specify `s` or `m`